pallets/automation-time/src/tests.rs

```diff
@@ -619,7 +619,7 @@ fn cancel_task_with_incorrect_schedule_as_will_fail() {
 	})
 }
 
-// Cancel task with incorrect schedule_as and non-existent taskid will fail
+// Cancel task with schedule_as and non-existent taskid will fail
```

Comment:
"will fail" is not specific enough. It’s better to declare the error here. Please use the full sentence in my previous comment.

Reply:
Fixed